Sources/AppcuesKit/Presentation/UI/Components/SelectToggleView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  SelectToggleView.swift
+//  AppcuesKit
+//
+//  Created by Matt on 2022-08-24.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+internal struct SelectToggleView<Content: View>: View {
+
+    enum Appearance {
+        case checkbox, radio
+    }
+
+    @Binding var selected: Bool
+    let appearance: Appearance
+    let controlPosition: ExperienceComponent.OptionSelectModel.ControlPosition?
+    @ViewBuilder var content: Content
+
+    @ViewBuilder var control: some View {
+        switch controlPosition {
+        case .leading:
+            HStack(spacing: 0) {
```

Comment:
checking understanding, so any spacing would be determined by the margin on the content item itself?

Reply:
Correct. Having baked-in styles that can't be changed seems like a generally bad idea.